src/main/java/com/kodilla/ecommercee/repository/OrderItemRepository.java

```diff
@@ -3,18 +3,11 @@
 import com.kodilla.ecommercee.domain.OrderItem;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
+
 import javax.transaction.Transactional;
-import java.util.List;
-import java.util.Optional;
 
 @Transactional
 @Repository
 public interface OrderItemRepository extends CrudRepository<OrderItem, Long> {
```

Comment:
To be agreed with others. You know my opinion :)

Reply:
For your peace of mind, we will do it your way.